### DISCUSSION TITLE
Flows Export

### DISCUSSION THREAD

**Discussion by rahulv1997** [postmanlabs/postman-flows#274] [Q&A] [answered]
How to Export Flows to run in newman

**Comment by ArminKugler** [comment]
tha t would also be my wish.

**Reply by rahulv1997** [reply]
what I'm asking is how to export that flows data 
if it's not import from that how would run that workflow

**Comment by samay-v** [comment, marked as answer]
Hey @rahulv1997, There was a similar thread which could help you out. https://github.com/postmanlabs/postman-flows/discussions/19#discussioncomment-1458902. TL;DR Running flows on the postman cloud environment is in our roadmap. It'll be our focus area in the coming quarters.

**Reply by rahulv1997** [reply]
okey Thanks you samay-v.....
we are eagerly waiting for that

**Comment by fbergeret95** [comment]
Hey! +1 to this thread. Even if the cloud environment is in the roadmap, a more offline approach would be much appreciated (exporting and importing files). Is this being considered?

**Reply by sanwat26** [reply]
Offline flows would have been nice. Not sure what will happen to created flow and efforts once the trial period ends or later if i chose to not opt for a plan, thats very restrictive considering the point that flows are still in beta stage.

**Comment by muddymind** [comment]
Flow exports is a must to me. It would allow me see the schema and create flow generators to cover many permutations of actions in huge flows. Otherwise I think It's more useful to create my own flows tool.
